scripts/generateCliDoc.js

```diff
@@ -195,18 +190,23 @@ function generateDoc() {
 
 }
 
-function splitString(command, temp, description) {
-	command = temp.split(/[^\w.,]\s(?=[A-Z])/)[0].trim();
-	description = temp.split(/[^\w.,]\s(?=[A-Z])/)[1].trim();
-	return { command, description };
+function splitString(temp) {
+	let command, description, details;
+
+	const match = temp.trim().split("  ").filter((s) => s).map((s) => s.trim());
+	if (match.length && match[match.length - 1].startsWith("[") && match[match.length - 1].endsWith("]")) {
+		details = match.splice(match.length - 1, 1)[0];
+	}
+	description = match.length ? match.splice(match.length - 1, 1)[0] : "";
+	command = match.length ? match[match.length - 1] : "";
```

Comment:
Not sure if this is easier to understand in general, but IMO to me it is, and it would also align with the previous lines by always pop'ing the elements:
```suggestion
	command = match.pop() || "";
```